packages/react/src/stories/ic-page-header.stories.mdx

```diff
@@ -36,6 +36,52 @@ import readme from "../../../web-components/src/components/ic-page-header/readme
 
 <Description markdown={readme} />
 
+### Playground
+
+export const defaultArgs = {
+  aligned: "left",
+  border: true,
+  heading: "Page Header Title",
+  reverseOrder: false,
+  size: "default",
+  sticky: false,
+  stickyDesktopOnly: false,
+  subheading: "Page Header Subtitle",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    name="Playground"
+    parameters={{ layout: "fullscreen" }}
+    argTypes={{
+      aligned: {
+        options: ["center", "full-width", "left"],
+        control: { type: "select" },
+      },
+      size: {
+        options: ["default", "small"],
+        control: { type: "select" },
+      },
+    }}
+  >
+    {(args) => (
+        <IcPageHeader
+          aligned={args.aligned}
+          border={args.border}
+          heading={args.heading}
+          reverseOrder={args.reverseOrder}
+          size={args.size}
+          sticky={args.sticky}
+          stickyDesktopOnly={args.stickyDesktopOnly}
+          subheading={args.subheading}
+        >
+          <IcStatusTag slot="heading-adornment" label="Beta" />
```

Comment:
Would be good to have some buttons in the action slot so that we can see the reverseOrder prop working

Reply:
Agreed, I'll add

Reply:
Added